app/controllers/test/components/notifications/icons_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Test::Components::Notifications::IconsController < ApplicationController
+  def show
+    @user = User.first
+  end
+
+  def update
+    NotificationsChannel.broadcast_to(
```

Comment:
Can this now use `NotificationsChannel .broadcast_changed`?

Reply:
Good idea. Adding this to our tracking issue.